modules/installation-osp-configuring-machine-network-cidr.adoc

```diff
@@ -0,0 +1,43 @@
+//Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-user.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installation-osp-configuring-machine-network-cidr_{context}"]
+= Configuring the machine network CIDR API and ingress virtual IP 
+
+The `machineNetwork` represents the {rh-openstack-first} network that will be used to connect all of the {product-version} cluster nodes. The `machineNetwork.CIDR` defines the IP range, in Classless Inter-Domain Routing (CIDR) notation.
+
+.Procedure 
+
+* Enter the following script to fill the values for `machineNetwork`, `apiVIPs`, `ingressVIPs`, and `controlPlanePort` into the `install-config.yaml` file for single-stack:
```

Comment:
This script is not only valid for for single-stack, but also dual-stack.
We can omit the single-stack mention.